packages/react-dom/src/events/DOMModernPluginEventSystem.js

```diff
@@ -214,23 +214,22 @@ function extractEvents(
   }
 }
 
-export const capturePhaseEvents: Set<DOMTopLevelEventType> = new Set([
-  TOP_FOCUS,
-  TOP_BLUR,
+// We should not delegate these events to the container, but rather
+// set them on the actual target element itself.
+export const nonDelegatedEvents: Set<DOMTopLevelEventType> = new Set([
   TOP_SCROLL,
   TOP_LOAD,
   TOP_ABORT,
   TOP_CANCEL,
   TOP_CLOSE,
-  TOP_INVALID,
-  TOP_ABORT,
```

Comment:
This confused me for a moment. But they're actually both still in the list. INVALID moved down and ABORT was listed twice.

Reply:
(This not actionable, just note for future readers)